epochStart/metachain/sovereignValidators.go

```diff
@@ -74,6 +77,29 @@ func (svic *sovereignValidatorInfoCreator) getMarshalledValidatorInfoTxs(miniBlo
 	return marshalledValidatorInfoTxs
 }
 
+// CreateValidatorInfoMiniBlocks creates the validatorInfo mini blocks according to the provided validatorInfo map
+func (svic *sovereignValidatorInfoCreator) CreateValidatorInfoMiniBlocks(validatorsInfo state.ShardValidatorsInfoMapHandler) (block.MiniBlockSlice, error) {
+	if validatorsInfo == nil {
```

Comment:
check.Ifnil ?

Reply:
`ShardValidatorsInfoMapHandler` does not implement `IsInterfaceNil`...
I would leave it like this...